bundle/config/transforms.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+const Paused = "PAUSED"
+const Unpaused = "UNPAUSED"
+
+type Transforms struct {
+	// Prefix to prepend to all resource names.
+	Prefix string `json:"prefix,omitempty"`
+
+	// DefaultPipelinesDevelopment is the default value for the development field of pipelines.
+	DefaultPipelinesDevelopment *bool `json:"default_pipelines_development,omitempty"`
+
+	// DefaultTriggerPauseStatus is the default value for the pause status of all triggers and schedules.
+	// Either config.Paused, config.Unpaused, or empty.
+	DefaultTriggerPauseStatus string `json:"default_trigger_pause_status,omitempty"`
```

Comment:
Are there other statuses you're thinking of? If not then this could also be a bool to toggle.